### PR TITLE
fix:"Hello Minikube" tutorial objective discrepancy: 'View Application Logs' mentioned but not covered .

### PR DESCRIPTION
**Pull request by MeenuyD**
One of the objectives stated at the beginning of the tutorial is to "View application logs", added in this pr
 https://kubernetes.io/docs/tutorials/hello-minikube/
#42984

### CODE REVIEW

File: content/en/docs/tutorials/hello-minikube.md [hunk 1]
```diff
@@ -142,6 +142,25 @@ recommended way to manage the creation and scaling of Pods.
     kubectl config view
     ```
 
+1. View application logs for a container in a pod.
+   
+   ```shell
+   # Return a snapshot of the logs from pod <pod-name>.
+   kubectl logs <pod-name>
+   ```
+
```
Comment (tengqm) [inline content/en/docs/tutorials/hello-minikube.md:151]: In a tutorial, you don't have to explain the syntax and then provide an example.
An example is pretty straightforward.
If you want to point the users to the command syntax, you can add a pointer at
the end of the page.

Comment (MeenuyD) [reply content/en/docs/tutorials/hello-minikube.md:151]: should I remove the explanation of syntax and example?

Comment (tengqm) [reply content/en/docs/tutorials/hello-minikube.md:151]: It should be fine or desirable to remove line 147-151.

**Comment by tengqm** [comment]
/label tide/merge-method-squash
/lgtm
/approve

Next time when you propose a PR, please always keep it at a single commit unless there is a good reason for multiple commits.

**Comment by MeenuyD** [comment]
Ok, I will do thanks for correcting me.

**Pull request merged**